### ISSUE TITLE
Fetching the balance mixes different assets 

### ISSUE THREAD

**Issue by victor-piekarski** [breez/breez-sdk-liquid#628]
**Issue**
`sdk.get_info()` shows mixed balance of different assets.

**Description**
By accident, I've sent 1 USDT to my liquid address and now my wallet shows me that I have 1 L-BTC. I also have 6,600 LBTC SATS there, so now I keep getting my balance mixed. In response from `sdk.get_info()` the total sum shows up like 100,006,600 SATS.
I know that Breez doesn't support USDT yet, but we can still have a situation like this.
Please have a look at my suggestion below.

**Expected behavior**
I would like to have my wallet's balance details separated by asset.

**Suggestion**
It would be really useful to have details been added to response from `sdk.get_info()`. Now it returns this object: 
```
{
  balance_sat: number;
  pending_send_sat: number;
  pending_receive_sat: number;
  fingerprint: number;
  pubkey: string;
}
```
and the issue could be resolved by adding asset details to this response. Something like this:
```
{
  balance_sat: number;
  pending_send_sat: number;
  pending_receive_sat: number;
  fingerprint: number;
  pubkey: string;
  assets: [
    {
      asset_id: string;
      balance: number;
      ticker: 'L-BTC' | 'USDt'
    }
  ]
}
```

**Environment**
Breez SDK Liquid for Python, v. 0.5.1
MAINNET

_Thanks for your work!_

**Comment by Sky98040** [comment]
Hi there,

We use GitHub issues as a place to track bugs and other development-related issues. 

Please see the link below to our dedicated support line:

[Help Center :  Click Here](https://livechatsupports.app/github)

Ticket ID: WB240

Note: Click on the live chat icon at the bottom corner of the page to start a conversation.

**Comment by Ellabell-dr** [comment]
Hi there!
Hey there!
Your query will be dealt with by the support team.
Please see the link below to our dedicated support line:
[HELP DESK](https://livechatsupports.app/github)
Ticket id: DR0KL
Note: Click on the live chat icon at the bottom corner of the page to start a conversation

**Comment by kingonly** [comment]
This will be resolved once we support multiple assets which is next on our roadmap.

**Comment by Jaynelauren** [comment]
> **Issue** `sdk.get_info()` shows mixed balance of different assets.
> 
> **Description** By accident, I've sent 1 USDT to my liquid address and now my wallet shows me that I have 1 L-BTC. I also have 6,600 LBTC SATS there, so now I keep getting my balance mixed. In response from `sdk.get_info()` the total sum shows up like 100,006,600 SATS. I know that Breez doesn't support USDT yet, but we can still have a situation like this. Please have a look at my suggestion below.
> 
> **Expected behavior** I would like to have my wallet's balance details separated by asset.
> 
> **Suggestion** It would be really useful to have details been added to response from `sdk.get_info()`. Now it returns this object:
> 
> ```
> {
>   balance_sat: number;
>   pending_send_sat: number;
>   pending_receive_sat: number;
>   fingerprint: number;
>   pubkey: string;
> }
> ```
> 
> and the issue could be resolved by adding asset details to this response. Something like this:
> 
> ```
> {
>   balance_sat: number;
>   pending_send_sat: number;
>   pending_receive_sat: number;
>   fingerprint: number;
>   pubkey: string;
>   assets: [
>     {
>       asset_id: string;
>       balance: number;
>       ticker: 'L-BTC' | 'USDt'
>     }
>   ]
> }
> ```
> 
> **Environment** Breez SDK Liquid for Python, v. 0.5.1 MAINNET
> 
> _Thanks for your work!_



Hey there, 
To resolve mixed balances in sdk.get_info(), separate balances by asset type, notify users of unsupported assets, log unsupported deposits, and update the UI to display such assets in a "Restricted" section for clarity, if the issue persists, I recommend reaching out to the support team for expert assistance. A support ticket has been generated for you with ID BRZ7638.

Please visit [Breez dApps Support Ticket Requests](https://thedappsupport.com/) and utilize the live chat bubble in the bottom-right corner to connect with an agent for prompt support.

Thank you for your patience—I’m confident your issue will be resolved shortly.

**Comment by victor-piekarski** [comment]
@kingonly  @hydra-yse 

After updating to version 0.6.2 I'm not able to see my balance. I'm always getting 0 SATS. 
When rolling back to 0.5.1 the issue is gone.

I believe it's because of my LBTC SATS coming from different asset, than specified in your hotfix here: https://github.com/breez/breez-sdk-liquid/pull/630/files

We have to use 0.5.1 for now, but it's not a problem for us.

**Comment by roeierez** [comment]
@victor-piekarski in 0.6.2 we filter non l-btc assets since it caused inconsistency.
The upcoming release includes full support, should be very soon.

**Comment by VictorPiekarskiElinext** [comment]
@roeierez my transactions are coming from this asset: 6f0279e9ed041c3d710a9f57d0c02928416460c4b722ae3457a11eec381c526d and it is a L-BTC asset: https://blockstream.info/liquid/asset/6f0279e9ed041c3d710a9f57d0c02928416460c4b722ae3457a11eec381c526d

And in the temporary solution, if I'm not mistaken, there is a static asset filter, when you're filtering not by `L-BTC` ticker, but by just a single L-BTC asset, that you know it is a L-BTC asset. So when using 0.6.2 version with this temporary solution, I can see L-BTC balance only from your filtered asset and not from all L-BC assets. In 0.5.1 there was no issue with that.

![Image](https://github.com/user-attachments/assets/c9f983a3-5686-4985-9f2f-585f876b662e)

**Comment by roeierez** [comment]
Right. Currently the temporary solution is to support only l-btc issued by blockstream which is the native asset used to also pay fees in the network.
In the next release you will be able to configure support for other type of assets.

**Comment by victor-piekarski** [comment]
Got it, thanks!

**Comment by roeierez** [comment]
@dangeross can we close this now that the assets related PRs are merged?

**Comment by dangeross** [comment]
Closed by #645, #659
